app/src/main/res/layout/item_profile.xml

```diff
@@ -7,41 +7,75 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="?attr/selectableItemBackground"
-    app:cardUseCompatPadding="true"
-    app:cardPreventCornerOverlap="false"
     app:cardCornerRadius="8dp"
-    app:cardBackgroundColor="@color/purple_200">
+    app:cardPreventCornerOverlap="false"
+    app:cardUseCompatPadding="true">
 
     <ImageView
         android:id="@+id/item_image"
-        android:contentDescription="SoundWaves"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:scaleType="centerCrop"/>
+        android:background="@color/blindly_blue"
+        android:contentDescription="SoundWaves"
+        android:scaleType="centerCrop" />
 
     <LinearLayout
-        android:orientation="vertical"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="bottom"
+        android:layout_height="707dp"
+        android:orientation="vertical"
         android:padding="16dp">
 
         <TextView
             android:id="@+id/item_name"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:textStyle="bold"
-            android:textColor="@android:color/white"
-            android:textSize="26sp"/>
+            android:textSize="26sp"
+            android:textStyle="bold" />
 
         <TextView
             android:id="@+id/item_age"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:textStyle="bold"
-            android:textColor="@android:color/white"
-            android:textSize="20sp"/>
+            android:layout_marginBottom="15dp"
+            android:textSize="20sp"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/item_description"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:justificationMode="inter_word"
```

Comment:
For some functions we had RequiresAPI annotation but not sure how having a API level 26 and higher might have an effect in a xml file. But I guess we can leave it if it's not replicable otherwise.